libraries/AP_OSD/AP_OSD.cpp

```diff
@@ -232,6 +232,15 @@ const AP_Param::GroupInfo AP_OSD::var_info[] = {
     AP_SUBGROUPINFO2(screen[2], "3_", 29, AP_OSD, AP_OSD_Screen),
     AP_SUBGROUPINFO2(screen[3], "4_", 30, AP_OSD, AP_OSD_Screen),
 #endif
+
+    // @Param: _TYPE2
+    // @DisplayName: OSD type 2
+    // @Description: OSD type 2. TXONLY makes the OSD parameter selection available to other modules even if there is no native OSD support on the board, for instance CRSF.
+    // @Values: 0:None,1:MAX7456,2:SITL,3:MSP,4:TXONLY,5:MSP_DISPLAYPORT
+    // @User: Standard
+    // @RebootRequired: True
+    AP_GROUPINFO("_TYPE2", 32, AP_OSD, osd_type2, 0),
```

Comment:
Has this been tested in SITL to make sure the ID is OK?

Reply:
if you mean has the SITL osd been tested....yes....it does work as 2nd type in SITL for its compatible cases

Reply:
Yes, all combinations but @Hwurzburg offered to retest